src/server/beego/v2.0/registry/catalog/catalog.go

```diff
@@ -0,0 +1,32 @@
+// Copyright Project Harbor Authors
```

Comment:
I personally don't think we should use `beego` in this pkg name because this `v2 handler` obviously has nothing to do with `beego` and can work with any http server.

Reply:
Removed `beego` directory